.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
Consider whether the `media/` directory should be excluded. If it contains important assets that need to be versioned, you might want to remove this line. Otherwise, if it contains generated or temporary files, it is appropriate to exclude it.